edsnlp/pipelines/misc/dates/patterns/atomic/months.py

```diff
@@ -1,5 +1,6 @@
 from edsnlp.utils.regex import make_pattern
 
+
```

Comment:
Apparently isort does not like this extra-line. Can you correct it, then we can finally merge! Sorry for the hassle...

```suggestion
```